subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -160,6 +159,13 @@ Supported Package Managers:
 * https://github.com/shyiko/jabba[Jabba]
 * https://sdkman.io/[SDKMAN!]
 
+[NOTE]
+====
+None of the toolchain types takes precedence over another. No location is prioritized over another, nor is any of the
```

Comment:
I would reverse the way this note is written: Instead of saying that there is no precedence by provider or location, directly indicate that there is a specific set of precedence rules and link.
Wdyt?